website/docs/tutorials/react-native.md

```diff
@@ -112,12 +112,14 @@ We're importing the default `i18n` object from `@lingui/core`. Read more about t
 
 Translating the heading is done. Now, let's translate the `title` prop in the `<Button title="mark messages as read" />` element. In this case, `Button` expects to receive a `string`, so we cannot use the [`Trans`](/docs/ref/macro.md#trans) macro here!
 
-The solution is to use the `t` macro together with the `i18n` object which we can obtain from the `useLingui` hook. We use the two like this to get a translated string:
+The solution is to use the `t` macro which we can obtain from the `useLingui` hook. We use it like this to get a translated string:
```

Comment:
`useLingui` macro?

Reply:
`useLingui` macro is a hook as well.  Could be used in the same context where runtime's `useLingui` used. 

This similar to Trans, which has two versions, macro and runtime. 

I think naming is fine, but documentation could be improved.